src/plugins/speech/structure_plugin.js

```diff
@@ -94,6 +147,7 @@ define( ['structure_custom' ], function( VisuDesign_Custom ) {
         selectedVoice = defaultVoice;
       }
       utterThis.voice = selectedVoice;
+      console.log("saying '%s' in voice %s", text, selectedVoice.name);
```

Comment:
Yes on purpose, should be kept till its more stable. Would be nice to have some sort of logger, where you can define a log-level, then we could leave those debug-messages where they are and set the log level to INFO for a release, whichs skips those messages.